### DISCUSSION TITLE
English:
I wanted to start a new Minecraft series on my youtube but i have no idea what my content should be 
___________________________________________
中文:
我想開始一個買塊的新系列在我的YouTube但是我想不到可以拍什麼

### DISCUSSION THREAD

**Discussion by zzxuebaizz87** [JoseDeFreitas/awesome-youtubers#71] [General]
English:
How do i do who can help me
___________________________________________
中文:
我該如何做,誰來救我

**Comment by JoseDeFreitas** [comment]
Hello @zzxuebaizz87! The objective of this repository is for people to find awesome tech YouTube channels that teach about technology topics. About your question, I don't think this is the right place to do it. I recommend you post your question in a forum or community so you can receive proper answers.
